app/views/fragments/javascriptFirstSteps.scala.html

```diff
@@ -108,3 +106,9 @@
 </script>
 
 
+<script type="text/javascript" async=1>
+;(function(p,l,o,w,i,n,g){if(!p[i]){p.GlobalSnowplowNamespace=p.GlobalSnowplowNamespace||[];
+p.GlobalSnowplowNamespace.push(i);p[i]=function(){(p[i].q=p[i].q||[]).push(arguments)
+};p[i].q=p[i].q||[];n=l.createElement(o);g=l.getElementsByTagName(o)[0];n.async=1;
+n.src=w;g.parentNode.insertBefore(n,g)}}(window,document,"script","//d1fc8wv8zag5ca.cloudfront.net/2.5.1/sp.js","snowplow_name_here"));
```

Comment:
Yeah, move it into vendor and then probably give it a name in `javascriptLaterSteps` like we do for omniture. You can then use something like `require(['js!snowplow', function() {})` to conditionally load it in. Probably worth initialising it in `analytics/setup` and guarding it with the `analyticsEnabled` check too.